src/Services/ResultService.php

```diff
@@ -29,37 +36,12 @@ public function setRepositoryManager(RepositoryManagerInterface $repositoryManag
         $this->repositoryManager = $repositoryManager;
     }
 
-    public function getAllUserTests(RequestInterface $request)
+    public function getAllUserTests(RequestInterface $request): array
     {
         $page = $request->getRequestParameters();
         $from = ($page['offset'] - 1) * 10;
```

Comment:
offset comes in the request?! either a naming issue, or business issue. also, what if $page['offset'] is not set?